book/07-extending-learners.qmd

```diff
@@ -5,47 +5,53 @@ library(mlr3)
-7. If applicable add references to the learner.
-8. Check [unit tests](#learner-test) and [paramtests](#learner-test) pass (these are automatically created).
-9. Run [cleaning functions](#cleaning)
-10. Open a [pull request](https://github.com/mlr-org/mlr3extralearners/pulls) with the new learner template.
-
-**(Do not copy/paste the code shown in this section. Use `r ref("create_learner()")` to start.)**
+6. If the learner has the property
 
-## Setting-up mlr3extralearners {#setup}
+    *  `"importance"` add the method `importance()`
+    * `"oob_error"` add the method `oob_error()`
+    * `"loglik"` add the method `loglik()`
+    * `"selected_features"` add the method `selected_features()`
+    *  `"hotstart_{forward, backward}"` add the private method `.hotstart()`
 
-In order to use the `r ref("mlr3extralearners::create_learner")` function you must have a local copy of the `r mlr3book::mlr_pkg("mlr3extralearners")` repository and must specify the correct path to the package. To do so, follow these steps:
+   More information on those methods can be found [here](#other-methods).
 
-1. [Fork](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/fork-a-repo) the [repository](https://github.com/mlr-org/mlr3extralearners)
-2. [Clone](https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/cloning-a-repository) a local copy of your forked repository.
+7. Fill out the missing parts in the learner's description.
+To add references you first need to create an entry in bibentries.R
+8. Check [unit tests](#learner-test) and [parameter tests](#learner-test) pass (these are automatically created).
+9. Run [cleaning functions](#cleaning)
+10. Open a [pull request](https://github.com/mlr-org/mlr3extralearners/pulls) with the "new learner" template.
 
-Then do one of:
+:::{.callout-warning}
+Do not copy/paste the code shown in this section. Use `r ref("create_learner()")` to start.
+:::
 
-* Open a new R session, call `library("mlr3extralearners")` (install if you haven't already), and then [run](#create-learner) `r ref("mlr3extralearners::create_learner")` with the `pkg` argument set as the path (the folder location) to the package directory.
-* Open a new R session, set your working directory as your newly cloned repository, run `r ref("devtools::load_all")`, and then [run](#create-learner) `r ref("mlr3extralearners::create_learner()")`, leaving `pkg = "."`.
-* In your newly cloned repository, open the R project, which will automatically set your working directory, run `r ref("devtools::load_all")`, and then run `r ref("mlr3extralearners::create_learner()")`, leaving `pkg = "."`.
+## Setting-up mlr3extralearners {#setup}
 
-We recommend the last option. It is also important that you are familiar with the three `devtools` commands:
+In order to use the `r ref("mlr3extralearners::create_learner")` function you must have mlr3extralearners installed.
```

Comment:
You can often just write "in order to use <function name>, [...]" without "the" and "function" within the sentence.

This should be obvious in general - I'd just leave it out.